### ISSUE TITLE
Twitch VOD Track 

### ISSUE THREAD

**Issue by Bl4cksus** [sorayuki/obs-multi-rtmp#366]
Hi,

Thank you for this plugin - it got me very close to my ideal setup!

One of my streaming platforms is twitch, and I would love to be able to also make use of the VOD Track feature of OBS with this plugin. 
I was hoping it would work when using the "get from OBS" dropdown, but it disregards the VOD Track option and only uses the main setting set in OBS.

Is adding this feature on the roadmap at all?

**Comment by nohaxjuscantcode** [comment]
It would also be amazing if we could choose different audio tracks for different streams as YouTube has much stricter copyright infringement policies.

**Comment by sorayuki** [comment]
You can specify standalone audio encoder for a target, and then use a different audio mixer id.
________________________________
发件人: nohaxjuscantcode ***@***.***>
发送时间: 2023年11月16日 2:41
收件人: sorayuki/obs-multi-rtmp ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [sorayuki/obs-multi-rtmp] Twitch VOD Track (Issue #366)


It would also be amazing if we could choose different audio tracks for different streams as YouTube has much stricter copyright infringement policies.

―

**Comment by geryyyyyy** [comment]
I understand that there is the possibility to choose from different audio channels that will be send with the stream.
However, I would also like to see the feature that @Bl4cksus described.

Twitch provides the possibility to stream two individual audio channels: One is for the actual livestream and one is used for the VOD. This is used for example to excluding the background music in the VODs.

One could use the build in OBS stream for Twitch to have this option. But if you want to stream to a second Twitch channel, you might need this option in your plugin.

In my case I would like to use it in combination with the option to choose a specific scene that will be streamed. My idea is to stream to Twitch/Youtube/Kick/..., but every stream should have a different overlay with chat/alters specifically for that platform.

**Comment by woopsmura** [comment]
Made an account just to second this request, I regularly stream on twitch and use the twitch vod track feature to have background music play to the Livestream, but not be in the VOD, it's very useful. If this feature could be usable in this wonderful plugin you've made, it would help myself and others very much.

**Comment by DanielHe212** [comment]
This is the one missing feature that would make this thing perfect for me.

**Comment by MajeoT** [comment]
Yeah please give us the option to stream to a "vod track" like is in the default OBS settings if you stream to twitch.

What this allows is for a "live" track to be sent to anyone watching live, so you can play alerts and other sounds ect that you might not want to be saved on the VOD itself. 

![1700839541045](https://github.com/sorayuki/obs-multi-rtmp/assets/4333223/73f0e955-15d8-4599-84d8-74c5d27407ad)

**Comment by StowGnar** [comment]
I, too, would really love this feature. Since I can only stream to FTL servers through OBS's main settings (YouNow, in my case), that means my main channel of Twitch would lose the VOD track capabilities I use every stream, if I were to make Twitch go through the Multiple Outputs plugin. Since YouNow has no interest in migrating to this decade by switching to RTMP, I currently can't stream to them even though I want to.

Adding OBS's dual Twitch audio track feature to Multiple Outputs would be the singular best feature that could be added.

**Comment by emmafaye** [comment]
Would love to see this as well!

**Comment by CortexReaver** [comment]
Yes please.

**Comment by BastianBN** [comment]
I only have "get from OBS" as encoding option in audio parameters, therefore I can't choose the track I want to encode to alternative sources.

![Image](https://github.com/user-attachments/assets/c0652892-b240-4bfa-914a-c5bb40585351)

**Comment by sorayuki** [comment]
are you running a pair of OBS/plugin with mismatched version? OBS changed the id of audio codec in a version.
________________________________

**Comment by BastianBN** [comment]
Yes I think, I updated everything yesterday, OBS to 31 and plugin to 0.6.0.1 but it didn't make the option appear
